fused-effects.cabal

```diff
@@ -157,6 +158,19 @@ test-suite test
     , transformers
 
 
+test-suite docs
+  import:      common
+  type:        exitcode-stdio-1.0
+  main-is:     README.lhs
+  ghc-options: -pgmL markdown-unlit
+  build-depends:
+      base
+    , fused-effects
+    , mtl           ^>= 2.2
+  build-tool-depends:
+      markdown-unlit:markdown-unlit ^>= 0.5
```

Comment:
Step six: we add a test component using `-pgmL markdown-unlit` to tell `ghc` to use that when dealing with `.lhs` files.